src/config.js

```diff
@@ -1,23 +1,28 @@
-const appConfig = {}
```

Comment:
I updated the initial assignment of `appConfig` to assign as much as it could upfront and then added the computed values afterwards.